scripts/buildProjects.ps1

```diff
@@ -32,6 +32,10 @@ function BuildServer ( $srcDir, $outDir, $target) {
         $commandArgs += "/p:UseAppHost=false"
     }
 
+    if ($ALLOW_ENCRYPTED_OVER_HTTP) {
+        $commandArgs += "/p:AllowEncryptedOverHttp=True"
```

Comment:
Actually we don't need this at all. Please see my last comment here :)

Reply:
OK, we had a little talk with Itay and decided to handle it this way:
- handle the RAVEN_BuildOptions change in csproj
- add a string switch called "$BuildOptions" defaulting to empty string
- if that's not null or empty, set $env:RAVEN_BuildOptions $BuildOptions
- add a string switch called "$ArtifactNameSuffix" defaulting to empty string
-  if that's not null or empty, set $env:ArtifactNameSuffix = $ArtifactNameSuffix
- where the filenames are set, instead of a targeted `if` let's do e.g. `$zipFile = "$($targetFilename)$($env:ArtifactNameSuffix).zip"`

Reply:
Done